src/commons/TestUtils.ts

```diff
@@ -27,4 +30,24 @@ export namespace TestUtils {
   export function extractNumbersFromString(str: string): string {
     return str.replace(/[^0-9]/g, '');
   }
+
+  /**
+   * Return object of data by file path
+   *  path File located in wdio config file,
+   */
+  export function getDataFromFile(): object {
+    // @ts-ignore
+    const dataFilename = browser.config.configDataFilePath;
+    const dataFilePath = path.resolve(process.cwd(), dataFilename);
+    return require(dataFilePath);
+  }
+
+  /**
+   * Get data by tag from e2eTestData file
+   * @param dataTag string
+   */
+  export function getData(dataTag: string = process.env.TEST_DATA_TAG): object {
+    const data: object = getDataFromFile();
+    return data && data[dataTag];
```

Comment:
it will always return data[dataTag] even if undefined

Reply:
@FelixZilber why do you suggest to change the logic here?
We want to make sure data has a value so we will not get an exception

Reply:
@nirgur the logic described by @AlexeyCL  did not fit the code...

Reply:
AFAIR the intention was what I described in my previous comment
@AlexeyCL can you please clarify?

Reply:
@nirgur yes, my explanation to Felix was incorrect. I'll revert it